robottelo/ui/hosts.py

```diff
@@ -61,6 +61,8 @@ def _configure_hosts(self, arch=None, custom_ptable=None, cv=None,
             self.wait_until_element(locators['host.ip']).send_keys(ip_addr)
         # Operating system tab
         if arch or os or media or ptable or custom_ptable or root_pwd:
+            self.scroll_page()
+            self.wait_for_ajax()
```

Comment:
So it makes more sense to add the wait_for_ajax call before the if  block to indicate that previous operations would require waiting for ajax. Being here, inside the if block and after the scroll_page means that scrolling would require the wait, not filling the previous value.